### PR TITLE
Fix remote integration for healthchecks

### PR DESCRIPTION
**Pull request by baude**
the one remaining test that is still skipped do to missing exec function

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: libpod/define/healthchecks.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+)
+
+// HealthCheckStatus represents the current state of a container
+type HealthCheckStatus int
+
+const (
+	// HealthCheckSuccess means the health worked
+	HealthCheckSuccess HealthCheckStatus = iota
+	// HealthCheckFailure means the health ran and failed
+	HealthCheckFailure HealthCheckStatus = iota
```
Comment (TomSweeneyRedHat) [inline libpod/define/healthchecks.go:21]: FWIW, you should be able to drop the ` = iota` from here on down

**Comment by TomSweeneyRedHat** [comment]
One suggestion for consideration otherwise
LGTM

**Pull request merged**